filter-api/src/main/java/org/hypertrace/agent/filter/FilterRegistry.java

```diff
@@ -23,8 +23,10 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.ServiceLoader;
+import org.hypertrace.agent.otel.extensions.config.HypertraceConfig;
```

Comment:
We can remove this. Then no circular dependency

Reply:
done. Thanks!